Polyfills/Canvas/Source/Image.cpp

```diff
@@ -0,0 +1,77 @@
+#include <bgfx/bgfx.h>
+#include <map>
+#include "Canvas.h"
+#include "Image.h"
+#include "Context.h"
+#include <functional>
+#include <sstream>
+#include <assert.h>
+#include <NativeEngine.h>
+
+namespace Babylon::Polyfills::Internal
+{
+    static constexpr auto JS_CONSTRUCTOR_NAME = "NativeCanvasImage";
+
+    void NativeCanvasImage::CreateInstance(Napi::Env env)
+    {
+        Napi::HandleScope scope{env};
+
+        Napi::Function func = DefineClass(
+            env,
+            JS_CONSTRUCTOR_NAME,
+            {
+                InstanceAccessor("width", &NativeCanvasImage::GetWidth, nullptr),
+                InstanceAccessor("height", &NativeCanvasImage::GetHeight, nullptr),
+                InstanceAccessor("naturalWidth", &NativeCanvasImage::GetNaturalWidth, nullptr),
+                InstanceAccessor("naturalHeight", &NativeCanvasImage::GetNaturalHeight, nullptr),
+                InstanceAccessor("src", &NativeCanvasImage::GetSrc, &NativeCanvasImage::SetSrc),
+                InstanceAccessor("onload", nullptr, &NativeCanvasImage::SetOnload),
```

Comment:
Setter and no getter seems weird, but I guess it's unlikely someone will get this.